PLAT-125200: Fix SpotlightRootDecorator to ignore `enter` key in touch mode (#2871)
enactjs/enact: CHANGELOG.md

The following is a curated list of changes in the Enact project, newest changes on the top.

## [unreleased]

### Fixed

> - `spotlight/SpotlightRootDecorator` to call preventDefault when `enter` key is pressed in touch mode

Reviewer: ```suggestion
- `spotlight/SpotlightRootDecorator` to ignore `enter` key in touch mode
```
